server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -0,0 +1,57 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import models from '../../../models';
+import { NotFound, Unauthorized } from '../../errors';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { Member } from '../object/Member';
+
+const memberMutations = {
+  editPublicMessage: {
+    type: new GraphQLNonNull(Member),
+    description: 'Edit the public message for the given Member of a Collective',
+    args: {
+      FromAccount: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account for the donating Collective',
+      },
+      ToAccount: {
```

Comment:
In other mutations for GQLV2, args are lowercase (ie https://github.com/opencollective/opencollective-api/blob/32f1ded723fbf354677c0fbca0eec3950d66e5e6/server/graphql/v2/mutation/AccountMutations.ts#L19). I think we should be consistent on this.

```suggestion
      fromAccount: {
        type: GraphQLNonNull(AccountReferenceInput),
        description: 'Reference to an account for the donating Collective',
      },
      toAccount: {
```